lang/java/avro/src/test/java/org/apache/avro/generic/TestGenericLogicalTypes.java

```diff
@@ -47,6 +51,8 @@
   public static void addDecimalAndUUID() {
```

Comment:
After adding the time conversions to this method the name (or the behavior) is really misleading.

Reply:
+1, will change the name